cli/connhelper/connhelper.go

```diff
@@ -79,3 +80,11 @@ func addSSHTimeout(sshFlags []string) []string {
 	}
 	return sshFlags
 }
+
+// Disable pseudo-terminal allocation.  This will prevent SSH from executing as a login shell
+func disablePseudoTerminalAllocation(sshFlags []string) []string {
+	if !strings.Contains(strings.Join(sshFlags, ""), "-T") {
```

Comment:
I don't get the `strings.Join()`. All args should be separate already in `[]string` iiuc. `slices.Contains()` should be enough.